docs/source/testing/mocking.md

```diff
@@ -3,13 +3,13 @@ title: Mocking
 description: Mock your GraphQL data based on a schema.
 ---
 
-The strongly-typed nature of a GraphQL API lends itself extremely well to mocking. This is an important part of a GraphQL-First development process, because it enables frontend developers to build out UI components and features without having to wait for a backend implementation.
+Mocking allows Apollo Server to return simulated data for GraphQL operations based on your server's schema. The strongly-typed nature of a GraphQL API lends itself extremely well to mocking, which is an important part of a GraphQL-first development process.  
 
-Even when the UI is already built, it can let you test your UI without waiting on slow database requests, or build out a component harness using a tool like [Storybook](https://storybook.js.org/) without needing to start a real GraphQL server.
+Mocking enables frontend developers to build out and test UI components and features without having to wait for a full backend implementation. Mocking is also valuable when using a UI tool like [Storybook](https://storybook.js.org/), because you won't need to start a real GraphQL server.
```

Comment:
Suggest "needing" instead of "having" (more immediately unambiguous for the reader)

Suggest "don't" instead of "won't" (we try to keep articles in the present tense unless describing a future step in a tutorial or similar)